app/editor/src/hooks/useLookupSync.ts

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
```

Comment:
Can eventually add the other lookups edited in the admin store

Reply:
Maybe this should be moved to the admin areas so it is updated as the user submits - versus when they load the list view..

Reply:
Yes it should be in the admin, but for now this will do.